### ISSUE TITLE
Unable to create study yml

### ISSUE THREAD

**Issue by alondhe** [OHDSI/Ulysses#11]
In my call to newOhdsiStudy() in a Linux-hosted Posit Workbench environment, I'm getting this error. 

```
ℹ Step 1: Creating R Project
✔ Setting active project to '/home/londheaj/oasis/o311'
ℹ Step 2: Adding OHDSI Study Project Folders
ℹ Step 3: Adding _study.yml file
Error in file(path, open = file_mode, encoding = "utf-8") : 
  cannot open the connection
In addition: Warning message:
In file(path, open = file_mode, encoding = "utf-8") :
  cannot open file 'oasis/o311/_study.yml': No such file or directory
```

**Comment by alondhe** [comment]
Seems to be fine when in the directory.

**Comment by mdlavallee92** [comment]
Could have been out of date code, fixed in v0.0.4
